metrics/web/test/base/presentation/widgets/icon_label_button_test.dart

```diff
@@ -183,13 +183,29 @@ void main() {
         ));
 
         final tappableArea = tester.widget<TappableArea>(
-          find.byType(TappableArea)
+          find.byType(TappableArea),
         );
         final actualCallback = tappableArea.onTap;
 
         expect(actualCallback, equals(testCallback));
       },
     );
+
+    testWidgets(
+      'applies tappable area to the row of the label and icon',
```

Comment:
We should use the double-quoted strings for the test descriptions according to the [dart code style](https://github.com/platform-platform/monorepo/blob/master/docs/10_dart_code_style.md#test-descriptions) document.